book/08-customizing-git/sections/config.asc

```diff
@@ -14,14 +14,15 @@ $ git config --global user.email johndoe@example.com
 Now you'll learn a few of the more interesting options that you can set in this manner to customize your Git usage.
 
 First, a quick review: Git uses a series of configuration files to determine non-default behavior that you may want.
-The first place Git looks for these values is in an `/etc/gitconfig` file, which contains values for every user on the system and all of their repositories.
+The first place Git looks for these values is in the system-wide `/etc/gitconfig` file, which contains settings applied to every user on the system and all of their repositories.
```

Comment:
How about "settings that are applied to"?

Reply:
Done.